science/projects/Ghazaleh Mahmoudi/01-main-content/main_content_detection.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    SVM_classifier(x_train, x_test, y_train, y_test)
</pre><p>sklearn's implementation provides a class weight argument, usefull for class balancing!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>